gooo/gooo.slide

```diff
@@ -0,0 +1,229 @@
+Object Orientation in Go
+What is object orientation anyway ? Does it matter ?
+21 Apr 2018
+Tags: software development, software design, paradigms
+
+Tiago Katcipis
+Programmer, Neoway
+tiagokatcipis@gmail.com
+https://katcipis.github.io
+@tiagokatcipis
+
+
+* Origins of Object Orientation
+
+- Every "new" thing usually has more than one influence
+- It is hard to pinpoint the origin of an idea
+- But people insist on being pedantic =D
+- Ranting like "Go is not OO"
+- So lets be pedantic too
+
+
+* Who is this guy ?
+
+.image ./img/kay.jpg
+
+
+* Who is this guy ?
+
+.image ./img/kay2.jpg _ 900
+
+
+* Alan Kay on Object Orientation
+
+At Utah sometime after Nov 66 when, influenced by Sketchpad, Simula, 
+the design for the ARPAnet, the Burroughs B5000, and my
+*background* in *Biology* and *Mathematics* , I thought of an architecture for 
+programming.
+
+It was probably in *1967* when someone asked me what I was 
+doing, and I said: "It's *object-oriented* programming".
+
+
+* Alan Kay on Object Orientation
+
+I thought of objects being like *biological* *cells* and/or individual 
+computers on a network, only able to *communicate* with *messages*.
+
+The term *"polymorphism"* was imposed much later (I think by Peter Wegner) and 
+it isn't quite valid, since it really comes from the nomenclature of 
+functions, and I wanted quite a bit more than functions.
+
+
+* Alan Kay on Object Orientation
+
+I didn't like the way Simula I or Simula 67 did inheritance.
+So I decided to leave out inheritance as a built-in 
+feature until I understood it better.
+
+
+* Alan Kay on Object Orientation
+
+I'm sorry that I long ago coined the term "objects" for this topic because
+it gets many people to focus on the lesser idea. The *big* *idea* is *"messaging"*
+
+The Japanese have a small word -- ma -- for "that which
+is in between" -- perhaps the nearest English equivalent is "interstitial"
+
+The key in making great and growable systems is much more to design how its
```

Comment:
maybe `The key to making`

Reply:
This is a quote from alan kay dude...do you want to correct ALAN KAY  ? =P

Reply:
hdudhhhdu, my bad, sorry for that